Client/Frontend/Browser/Tabs/RemoteTabs/RemoteTabsPanel.swift

```diff
@@ -51,12 +68,25 @@ class RemoteTabsPanel: UIViewController,
                                        object: nil)
     }
 
+    @objc
+    func notificationReceived(_ notification: Notification) {
```

Comment:
> We can remove the notification logic this will be done in the Profile middleware

From initial team chats it sounded like we don't yet have a way of implementing this directly in Redux; AFAIU, the event queue that @OrlaM mentioned previously was the original plan to handle these.

I'm happy to remove them anyway, though I'm not sure if we'll have the mechanisms required to support these hooks via Redux in the next few days as additional wiring is added to the sync tab, which means I may have to add them back in, or perhaps we can sync up on an alternative solution.

Reply:
We could have the exact same logic in the Middleware and trigger the actions that will update the ViewController and views subscribed. Let's leave it for now and we can move it once we work in the Middleware class, this will be a workaround in case the event queue work is not ready.

Reply:
@yoanarios Ok sounds good; if this gets too messy LMK and I'm happy to clean this up further or just remove it until the middleware is in place.